tempodb/wal/local_block.go

```diff
@@ -19,16 +19,16 @@ const nameFlushed = "flushed"
 // LocalBlock is a block stored in a local storage.  It can be searched and flushed to a remote backend, and
 // permanently tracks the flushed time with a special file in the block
 type LocalBlock struct {
```

Comment:
ultimately i think LocalBlock will need to be an interface like BackendBlock and be either parquet or the current format or whatever so it can be directly shipped to object storage.

The AppendBlock can remain in the current format for the foreseeable future and we can do the translation when we "cut" the head block.